### PR TITLE
Organization Folder

### PR DESCRIPTION
**Pull request by galuskodrobova**
Create Organization Folder
Rename Organization Folder
Delete Organization Folder

### CODE REVIEW

**Review by tatauch — APPROVED** [review]

File: src/test/java/OrganizationFolderTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import runner.BaseTest;
+import java.util.List;
+
+public class OrganizationFolderTest extends BaseTest {
+
+    private final String VALID_VALUE_FOR_NAME1 = "Organization Test";
+    private final String VALID_VALUE_FOR_NAME2 = "Test";
+    private final String VALID_VALUE_FOR_NAME3 = "Organization TestTest";
+
+
```
Comment (anonymoustuna) [inline src/test/java/OrganizationFolderTest.java:15]: No need in this empty line.

File: src/test/java/OrganizationFolderTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    public void getHomePage() {
+        getDriver().findElement(By.xpath("//a[text()='Dashboard']")).click();
+    }
+
```
Comment (anonymoustuna) [inline src/test/java/OrganizationFolderTest.java:19]: Empty line, you might delete it.

File: src/test/java/OrganizationFolderTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        getDriver().findElement(By.xpath("//span[@class ='label'][text()='Organization Folder']")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.id("yui-gen17-button")).click();
+        getHomePage();
+
+        Assert.assertEquals
+                (getDriver().findElement(By.xpath("//a[text()='Organization Test']"))
+                                .getText(),"Organization Test");
+    }
+
```
Comment (anonymoustuna) [inline src/test/java/OrganizationFolderTest.java:38]: You might delete this empty line as well.

File: src/test/java/OrganizationFolderTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        String expectedResult = VALID_VALUE_FOR_NAME3;
+        getDriver().findElement(By.xpath("//a[text()='Organization Test']")).click();
+        getDriver().findElement(By.xpath("//span[contains(text(),'Rename')]")).click();
+        getDriver().findElement(By.name("newName")).sendKeys(VALID_VALUE_FOR_NAME2);
+        getDriver().findElement(By.id("yui-gen1-button")).click();
+        String actualResult = getDriver().findElement(By.xpath("//a[text()='Organization TestTest']")).getText();
+
+        Assert.assertEquals(actualResult,expectedResult);
+    }
+
```
Comment (anonymoustuna) [inline src/test/java/OrganizationFolderTest.java:51]: Empty line.

File: src/test/java/OrganizationFolderTest.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        List<WebElement> tableOnDashboard =
+                getDriver().findElements(By.xpath("//table[@id='projectstatus']/tbody/tr/td/a"));
+        for (WebElement item : tableOnDashboard){
+            if (item.getText().contains(VALID_VALUE_FOR_NAME1)) {
+                getDriver().findElement(By.xpath("//a[text()='Organization TestTest']")).click();
+                getDriver().findElement(By.linkText("Delete Organization Folder")).click();
+                getDriver().findElement(By.id("yui-gen1-button")).click();
+                break;
+            }
+        }
```
Comment (anonymoustuna) [inline src/test/java/OrganizationFolderTest.java:65]: You might add space after.

**Review by anonymoustuna — APPROVED** [review]

**Pull request merged**